.devcontainer/docker-compose.yml

```diff
@@ -8,6 +8,8 @@ services:
     build:
       context: .
       dockerfile: .devcontainer/Dockerfile
+    env_file:
```

Comment:
Oh cool, I didn't know about `env_file:`

I think that we will want `initializeCommand: "cp .env.local.sample .env.local"` in the devcontainer.json -- .env.local wouldn't exist yet and be gitignored...

Reply:
i tested "initializeCommand: "cp .env.local.sample .env.local""  and it looks like it runs locally , not in the docker container , which is an issue on Windows.

ill move it to the dockerfile  and test

Reply:
adding the cp in dockerfile doesnt work , because at that stage the workspace dir is not  mapped 

putting it in devcontainer.json(onCreateCommand, etc....) also does not work , because docker-compose runs before and fails if .env.local is not present